modules/cnv-storage-wizard-fields-web.adoc

```diff
@@ -20,9 +20,9 @@
 |Size, in GiB, of the disk.
 
 |Interface
-|Name of the interface.
+|Type of disk device such as *virtIO*, *SATA*, and *SCSI*.
```

Comment:
I still think it should be something like "... disk device. Supported interfaces are ..."
Having "such as" implies that there are alternatives not listed.

Reply:
Agree. Made the change - please review. Thanks!